packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -827,36 +819,6 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		}
 	}
 
-	/**
-	 * This is called to update blobs whose routes are tombstones.
-	 *
-	 * A Tombstoned object has been unreferenced long enough that GC knows it won't be referenced again.
-	 * Tombstoned objects are eventually deleted by GC.
-	 *
-	 * @param tombstonedRoutes - The routes of blob nodes that are tombstones.
-	 */
-	public updateTombstonedRoutes(tombstonedRoutes: readonly string[]) {
```

Comment:
No test coverage it seems?  Good riddance!

Reply:
We have end-to-end tests for tombstone attachment blobs. They still work as expected so all good :-).